composer.json

```diff
@@ -11,9 +11,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "php": ">=7.1.0",
         "silverstripe/vendor-plugin": "^1.0",
         "silverstripe/elemental-fileblock": "^2@dev",
-        "silverstripe/cms": "^4.3"
+        "silverstripe/cms": "^4.3",
+        "silverstripe/admin": "^1.7@dev"
```

Comment:
why is admin at 1.7 (which doesn't exist) and cms is at 4.3?

Reply:
When the next version is release it will require admin 1.7. I would only bump the cms dependency if I had a PR that was referencing an API that wasn't available in cms 4.3.

In practice, it's probably impossible to install CMS 4.3 along with admin 1.7. But I'm quite happy to let composer sort this out.